doc/guide/guide-overview.rst

```diff
@@ -4,22 +4,14 @@
 Guide Overview
 ******************
 
-The goal of this guide is to introduce you to the basic structures and functions that make up QuTiP. This guide is divided up into several sections, each highlighting a specific set of functionalities. In combination with the examples that can be found on the project web page `https://qutip.org/tutorials.html <https://qutip.org/tutorials.html>`_, this guide should provide a more or less complete overview. In addition, the :ref:`apidoc` for each function is located at the end of this guide.
+The goal of this guide is to introduce you to the basic structures and functions that make up QuTiP. This guide is divided up into several sections, each highlighting a specific set of functionalities. In combination with the examples that can be found on the project web page `https://qutip.org/tutorials.html <https://qutip.org/tutorials.html>`_, this guide should provide a more or less complete overview. In addition, the :ref:`apidoc` for each function is located below.
```

Comment:
I do not think "below" is strictly correct here. However, "end of this guide" does not seem correct either as It seems to suggest that the information is within this chapter, but is not. I suggest rephrasing the paragraph a bit. I changed it to be more direct:
```suggestion
The goal of this guide is to introduce the basic structures and functions in QuTiP. This guide is divided up into several sections, each highlighting a specific set of functionalities. In combination with the examples that can be found on the project web page `https://qutip.org/tutorials.html <https://qutip.org/tutorials.html>`_, this guide should provide a more or less complete overview of QuTiP. We also provide the API documentation in :ref:`apidoc`.
```